.env_example

```diff
@@ -1,6 +1,35 @@
-CONNECTION_STRING_POSTGRESQL="user=xxxxx password=xxxxxx dbname=xxxxxx port=xxxxxx sslmode=disable"
+IS_PROD="true"
+CONNECTION_STRING_POSTGRESQL="user=xxxx password=xxxxx dbname=xxxxxx port=5433 sslmode=disable"
+CONNECTION_STRING_POSTGRESQL_PROD="host=/cloudsql/xxxxxx:xxxxxx user=xxxxxxx password=xxxxxxxx dbname=xxxxxx sslmode=disable"
+
 GO_API_PORT=3000
+GO_API_PORT_PROD=8080
+
 AE_NODE_URL="http://localhost:3001"
-SECRET_KEY="xxxxxxxxxxxxx" 
-CONTRACT_SOURCE_URL_GIT_RAW="https://raw.githubusercontent.com/LimeChain/aepp-token-migration-smart-contract/master/contracts/TokenMigration.aes" 
-AE_CONTRACT_TOKEN_MIGRATION_ADDRESS="ct_nvYGScRcwHWurpUWg9qPyWNQKxnSxG55zXoWSQigMSYSa8g51"
\ No newline at end of file
+AE_NODE_URL_PROD="xxxxxxx"
+
+SECRET_KEY="AE_SECRET_KEY" 
+CONTRACT_SOURCE_URL_GIT_RAW="https://raw.githubusercontent.com/your_contractg_url.aes"
+
+AE_CONTRACT_TOKEN_MIGRATION_ADDRESS="smart_contract_address"
+
+AE_COMPILER_URL="http://localhost:3080"
+AE_COMPILER_URL_PROD="xxxxxxx"
+
+AE_NETWORK_ID="ae_devnet"
+AE_NETWORK_ID_PROD="ae_devnet"
+# AE_NETWORK_ID_PROD="ae_uat"
+
+AE_BACKEND="fate"
+AE_ABI_VERSION=3
+
+BACKENDLESS_LOGIN="xxxxxx"
```

Comment:
again, do we need this in the exampple?

Reply:
I removed it in another PR, now removed in this ;)